src/binding.cc

```diff
@@ -2056,6 +2056,28 @@ void v8_inspector__V8Inspector__contextCreated(
       ptr_to_local(&context), contextGroupId, humanReadableName));
 }
 
+unsigned v8_inspector__V8Inspector__exceptionThrown(
+    v8_inspector::V8Inspector* self, const v8::Context& context,
+    v8_inspector::StringView message, const v8::Value& exception,
+    v8_inspector::StringView detailed_message, v8_inspector::StringView url,
+    unsigned line_number, unsigned column_number,
+    v8_inspector::V8StackTrace* stack_trace, int script_id) {
+  return self->exceptionThrown(ptr_to_local(&context), message,
+    ptr_to_local(&exception), detailed_message, url, line_number,
+    column_number, static_cast<std::unique_ptr<v8_inspector::V8StackTrace>>(stack_trace), 
```

Comment:
I'm afraid this "consumes" the `V8StackTrace` object.
Is it possible to call `exception_thrown()` in rust twice without crashing?